File: backend/src/main/java/org/jdk/project/dto/config/ComplaintUpsertRequest.java
package org.jdk.project.dto.config;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ComplaintUpsertRequest {
  private Long loungeId;
  private String userName;
  private String cardService;
  private String cardNo;
  private String complaintContent;
  private String complaintFeedback;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/ComplaintUpsertRequest.kt
package org.jdk.project.dto.config

data class ComplaintUpsertRequest(
    var loungeId: Long? = null,
    var userName: String? = null,
    var cardService: String? = null,
    var cardNo: String? = null,
    var complaintContent: String? = null,
    var complaintFeedback: String? = null,
)
